fix(#248): tried to auto-install dep

CedricGatay opened the pull request:
We now auto install dependencies thanks to https://github.com/JakeWharton/sdk-manager-plugin

CedricGatay committed:
fix(#248): tried to auto-install dep

We now auto install dependencies thanks to https://github.com/JakeWharton/sdk-manager-plugin

CedricGatay commented:
It seems ok, as the build logs outputs now
`Build tools 23.0.2 missing. Downloading...` 

Without this the build associated with this PR would have fail.

Once merged, one need to edit cloudbees jobs to remove the first android sdk steps (on master first to check if this is working, then on pr builder later maybe). 

@team, feel free to mention me, I'll do it.

JorisPotier commented:
It seems that a clean jenkins job without the android sdk steps as mentioned above cannot retrieve the build tools by himself, even if the logs are saying that it is downloading...

```
(...)
Build tools 23.0.2 missing. Downloading...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':library'.
> Failed to notify project evaluation listener.
   > failed to find Build Tools revision 23.0.2
   > failed to find Build Tools revision 23.0.2
```

The job is available on our cloudbees if you want.

CedricGatay committed:
fix(248): set the plugin higher

Tried to position the plugin higher in the build hierarchy

CedricGatay committed:
fix(248) : problem with build tools

Build tools 23.0.2 cause issue with building, tried a fork of jake wharton's tool to specify minSdkVersion
